docs/source/index.rst

```diff
@@ -27,8 +27,15 @@ configurations.
 .. _Cartographer: https://github.com/googlecartographer/cartographer
 .. _SLAM: https://en.wikipedia.org/wiki/Simultaneous_localization_and_mapping
 
+Getting started
+===============
+
+The easiest way to get started with Cartographer is to use `ROS
```

Comment:
I would switch this around: Cartographer is a standalone, framework agnostice c++ library. To see it in action quickly ...